.github/workflows/checks-quick.yml

```diff
@@ -128,7 +128,7 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.0 (22. Sep 2023)
       - name: install python deps
         run: |
-          sudo apt-get update && sudo apt-get install -y python3-pip python3
+          # sudo apt-get update && sudo apt-get install -y python3-pip python3
```

Comment:
Deliberate?

Reply:
I checked if it works without it and the removed this line since python and pip are presented in the ci image.